GlimpseImputationPipeline/Glimpse2Imputation.wdl

```diff
@@ -165,28 +171,18 @@ task GlimpsePhase {
         cram_index_paths=( ~{sep=" " cram_indices} )
         sample_ids=( ~{sep=" " sample_ids} )
 
-        if ~{if defined(cram_indices) then "true" else "false"}; then
-            seq_cache_populate.pl -root ./ref/cache ~{fasta}
-            export REF_PATH=:
-            export REF_CACHE=./ref/cache/%2s/%2s/%s
-        
-            chunk_region=$(echo "~{reference_chunk}"|sed 's/^.*chr/chr/'|sed 's/\.bin//'|sed 's/_/:/1'|sed 's/_/-/1')
+        duplicate_cram_filenames=$(printf "%s\n" "${cram_paths[@]}" | xargs -I {} basename {} | sort | uniq -d)
```

Comment:
Given that we now explicitly define the indices, are duplicate basenames still an issue? Are the index file still localized?

(It's possible that you're going to answer this question in your deep dive in 30 minutes, haha)

Reply:
the indices are still optional inputs, so I think we still want to check for duplicates.

Reply:
At this point, do we just want to require the indices? I don't know what happens when no index is provided (is one created? Or is just the file parsed until it encounters the region?), but either way it feels inefficient to do that 800 times in parallel

Reply:
Yeah I see 3 options here, all of them are fine
1) Leave it as is
1) Add a check in bash that, in case of CRAM input, verifies if indices are defined
1) Remove VCF input option and make CRAM and CRAI required

I'm going to approve this PR, feel free to add any of these changes or not.